packages/zowe-explorer-api/src/profiles/ZoweExplorerZosmfApi.ts

```diff
@@ -329,8 +329,12 @@ export class ZosmfJesApi extends ZosmfApiCommon implements ZoweExplorerApi.IJes
  */
 // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
 export class ZosmfCommandApi extends ZosmfApiCommon implements ZoweExplorerApi.ICommand {
-    public async issueTsoCommand(command: string, acctNum: string): Promise<zowe.IIssueResponse> {
-        return await zowe.IssueTso.issueTsoCommand(this.getSession(), acctNum, command);
+    public async issueTsoCommand(
+        command: string,
+        acctNum: string,
+        startParams: zowe.IStartTsoParms
+    ): Promise<zowe.IIssueResponse> {
+        return await zowe.IssueTso.issueTsoCommand(this.getSession(), acctNum, command, startParams);
```

Comment:
here, with passing only startParams we could make the call 
`return await zowe.IssueTso.issueTsoCommand(this.getSession(), startParams.account, command, startParams);`